module2/homework/transformContainers/transformContainers.hpp

```diff
@@ -0,0 +1,7 @@
+#pragma once
+#include <deque>
+#include <list>
+#include <string>
+#include <map>
```

Comment:
Firstly you should add newline between pragma and standard library includes.
Secondly includes in one paragraph should be sorted alphabetically.